### DISCUSSION TITLE
ScriptReference/Vector3.Lerp

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#614] [Notes]
# ScriptReference/Vector3.Lerp

https://docs.unity3d.com/ScriptReference/Vector3.Lerp.html

**Comment by EliasADU** [comment]
If you're trying to lerp towards a moving target, as has been a common use case for me, consider something like George Foot's "SuperSmoothLerp" function (from https://forum.unity.com/threads/how-to-smooth-damp-towards-a-moving-target-without-causing-jitter-in-the-movement.130920)

    Vector3 SuperSmoothLerp(Vector3 x0, Vector3 y0, Vector3 yt, float t, float k)
    {
        Vector3 f = x0 - y0 + (yt - y0) / (k * t);
        return yt - (yt - y0) / (k*t) + f * Mathf.Exp(-k*t);
    }
